app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -286,24 +304,72 @@ public void failure(RetrofitError error) {
     private void initChapters(@NonNull ArrayList<Chapter> chapters) {
         addChapters(chapters);
 
-        mViewPagerAdapter = new ChapterFragmentPagerAdapter(this,
-                getSupportFragmentManager(), selectedChapterId);
+        mViewPagerAdapter = new ChapterFragmentPagerAdapter(
+                this,
+                getSupportFragmentManager(), selectedChapterId
+        );
         mViewPager.setAdapter(mViewPagerAdapter);
         mViewPager.setOffscreenPageLimit(3);
         updateSelectionfor(selectedChapterId);
 
-        mTabLayout.setupWithViewPager(mViewPager);
+        recordStartPageView();
```

Comment:
I have thought about it when I saw your comment here. Thank you for reminding. 
I am actually not sure. 

When I think normal webpages and SEO, main url's are always indexed more. I think it makes sense.